indico/modules/events/registration/forms.py

```diff
@@ -423,3 +424,33 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    """Form to create a new registration tag."""
+
+    title = StringField(_('Title'), [DataRequired()])
+    color = StringField(_('Color'), [DataRequired()], widget=ColorInput())
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event', None)
```

Comment:
This isn't optional (and in fact it's currently breaking edits since it's missing when instantiating the form for editing)

```suggestion
        self.event = kwargs.pop('event')
```